whylog_vim/tests/tests_output_formater/tests_output_agregator.py

```diff
@@ -0,0 +1,35 @@
+from whylog_vim.output_formater.output_agregator import OutputAgregator
+
+
+def tests_basic_output_agregator():
+    class TestFunctions():
+        def click_function(self, param1, param2):
```

Comment:
Please rename to be more specific. Later in code I don't remember what does `click_function`. Maybe `addition_on_click`?